Dockerfile

```diff
@@ -11,12 +11,14 @@ WORKDIR /usr/src/app
 # rebuild image when package.json or lock has changed
 COPY package*.json ./
 
-# install dependencies
-RUN npm ci --only=production
-
 # add project artifacts to docker image
 ADD . .
 
+# clean and reinstall dependencies, otherwise we'll get
+# incorrect binaries if the container and host are not
+# same OS and arch
+RUN rm -fr node_modules && npm ci --omit=dev
```

Comment:
@StarpTech please verify

Reply:
We don't need to delete it here. We will never copy node_modules https://github.com/wundergraph/docker/blob/main/.dockerignore#L2

Reply:
In my tests it does get copied anyway, using `Docker version 20.10.17, build 100c701`

Reply:
It shouldn't. We use it in several other places in the cloud.